extensions/applicationinsights-dependencies-js/Tests/Unit/src/ajax.tests.ts

```diff
@@ -1,4 +1,4 @@
-﻿import { SinonStub } from "sinon";
+import { SinonStub } from "sinon";
```

Comment:
hmm.. this change wasn't in what I staged, and my editor is telling me it's not actually changed.  I'm guessing this is a CR/LF issue I need to sort out.

Reply:
ok, got it fixed by using `git` instead of VSCode to do the interactive add to ignore that hunk.